.ci/run.sh

```diff
@@ -49,6 +49,10 @@ case "${CI_JOB}" in
 		echo "INFO: Running complete e2e kubernetes tests"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make kubernetes-e2e"
 		;;
+	"FIRECRACKER")
+		echo "INFO: Containerd checks"
```

Comment:
In these `case` branches, there are 3 `echo "INFO: Containerd checks"`, I'd suggest echo more distinct info by adding the case value(clh or firecracker or qemu) into it.